src/App.jsx

```diff
@@ -14,70 +15,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedItem, setSelectedItem] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedItem ? `${selectedItem} is selected` : 'No goods selected'}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {selectedItem && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedItem('')}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
+            <tr
+              key={item}
+              data-cy="Good"
+              className={
+                selectedItem === item
+                  ? 'has-background-success-light'
+                  : ''
+              }
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  onClick={() => (selectedItem === item
+                    ? setSelectedItem('')
+                    : setSelectedItem(item))
+                  }
+                  data-cy={
+                    selectedItem === item
+                      ? 'RemoveButton'
+                      : 'AddButton'
+                  }
+                  type="button"
+                  className={
+                    selectedItem === item
```

Comment:
Create a helper function for this checking `selectedItem === item` and use it everywhere